SPECS/rpm/rpm.spec

```diff
@@ -1,7 +1,7 @@
 Summary:        Package manager
 Name:           rpm
 Version:        4.17.0
-Release:        5%{?dist}
+Release:        6%{?dist}
```

Comment:
Can you bump to release 7, since the package is already at release 6 in the main branch